recipes/newrelic/infrastructure/ohi/elasticsearch/debian.yml

```diff
@@ -51,8 +51,12 @@ inputVars:
   - name: "NR_CLI_API_CA_BUNDLE_FILE"
     prompt: "SSL Certificate File (if applicable)"
 
-install:
+preInstall:
+  info: |2
+      To capture data from the Elasticsearch integration, you'll first need to meet these prerequisites:
+      - Elasticsearch version 5.x through 7.x
```

Comment:
We shouldn't exclude more recent versions that the ohi is likely going to support in the future.